src/frontend/Navigation/AppNavigator.tsx

```diff
@@ -33,10 +26,7 @@ export const AppNavigator = () => {
       initialRouteName="Home"
```

Comment:
Reading through the rest of the implementation I guess probably not, because we need the Home screen on the stack so that we can "goBack" to it when we auth.